packages/carbon-graphs/src/js/controls/Scatter/helpers/helpers.js

```diff
@@ -134,6 +134,14 @@ const dataPointActionHandler = (value, index, target) => {
  * @returns {undefined} - returns nothing
  */
 const translateScatterGraph = (scale, canvasSVG, config) => {
+  canvasSVG
+    .select(`.${styles.currentPointsGroup}`)
+    .attr(
+      'transform',
+      `translate(${getXAxisXPosition(
+        config,
+      )},${calculateVerticalPadding(config)})`,
+    );
```

Comment:
will abstracting this into its own helper do more harm/good? i'm not super familiar with this code, but i do notice we execute this logic 3 times. the only thing that differs is the style. probably come up with better names for the helper and params

```js
const helper = (style, config) => (
  canvasSVG
    .select(`.${styles}.${style}`)
    .attr(
      'transform',
      `translate(${getXAxisXPosition(config)},${calculateVerticalPadding(config)})`,
    );
);

Reply:
That sounds like a good idea, updated here: fc46388c575dc0344b6c92180408c47b8a0c5dc5